tableau-scripts/tableau-pub.py

```diff
@@ -1,9 +1,12 @@
+#!/bin/python
```

Comment:
would suggest:
/usr/bin/env python
in case python is not installed in /bin (if this script is run on a different server)

Reply:
Thx for the suggestion.